system/proposal-system.md

```diff
@@ -816,6 +816,117 @@ None.
 
 The councilor reward is set to `new_councilor_reward`.
 
+### Update Global NFT Limit
+
+#### Parameters
+
+| Name   | Description                                               |
+| ------ | --------------------------------------------------------- |
+| `nft_limit_period` | The type of period for which the limit should be updated. |
+| `limit` | The value for the limit. |
+
+#### Constants
+
+| Constant             | Value     |
+| -------------------- | --------- |
+| `DECIDING_PERIOD`    | `fill-in` |
+| `GRACE_PERIOD`       | `fill-in` |
+| `APPROVAL_QUORUM`    | `fill-in` |
+| `APPROVAL_THRESHOLD` | `fill-in` |
+| `SLASHING_QUORUM`    | `fill-in` |
+| `SLASHING_THRESHOLD` | `fill-in` |
+| `PROPOSAL_STAKE`     | `fill-in` |
+| `CONSTITUTIONALITY`  | `fill-in` |
+
+#### Creation Conditions
+
+#### Execution Conditions
+
+Global limit value for specified period is updated
+
+### Create Blog Post
```

Comment:
Why is all this blog post stuff here? that was dropped from the handbook? did you pull from upstream repo first?

Reply:
it should be removed now, I have also format on save on my editor so you see all the * bullet points becoming - bullet points.